ha/util/conf_store.py

```diff
@@ -98,9 +98,10 @@ def set_cluster_cardinality(self, index):
 
         data_pods = ConftStoreSearch.get_data_pods(index)
         server_pods = ConftStoreSearch.get_server_pods(index)
+        control_pods = ConftStoreSearch.get_control_pods(index)
 
         # Combine the lists data_pods, server_pod and find unique machine ids
```

Comment:
Please update comment for control pod as well.

Reply:
Yes. I got it. I am saying, for line 103, update the comment for control pod as well.

Reply:
Done.

Reply:
Thanks.